sites/partners/pages/index.tsx

```diff
@@ -9,7 +9,15 @@ import { GridOptions } from "ag-grid-community"
 import { useListingsData } from "../lib/hooks"
 import Layout from "../layouts"
 import { MetaTags } from "../src/MetaTags"
-import { Router, useRouter } from "next/router"
+import { useRouter } from "next/router"
+
+import Select from "react-select"
+
+const options = [
```

Comment:
Filling this array could be a place for using `getStaticProps()`, since it'll be the same for every user.

We'd need to add a backend endpoint for the data.

Alternatively, there aren't that many neighborhoods right? So we could just hardcode them.

Reply:
Yes lets discuss together, as I just hardcoded the options for now, but dynamically calling them might be a better solution longterm for other cities to leverage.